nerds-catalog.html

```diff
@@ -110,164 +115,189 @@ <h2>Особенности:</h2>
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-mishka.jpg"
+              alt="Макет Mishka"
+            />
+            <div class="product-item-hover">
+              <h4>MISHKA</h4>
+              <p>Интернет-магазин вязаных игрушек</p>
+              <div>
+                <a href="#" aria-label="Цена" class="button">9900 РУБ.</a>
+              </div>
+            </div>
+          </article>
 
-        <article class="product-item">
-          <h3 class="visually-hidden">Макет A Plus</h3>
-          <img
-            src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
-            alt="Хэдер браузера"
-          />
-          <img
-            src="../Nerds.web/img/catalog_img/img-aplus.jpg"
-            alt="Макет A Plus"
-          />
-          <!-- <div class="product-item-hover">
-					<h4>A PLUS</h4>
-					<p>Лендинг курсов повышения квалификации</p>
-					<div><a href="#" aria-label="Цена">9900 РУБ.</a></div>
-				</div> -->
-        </article>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет A Plus</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-aplus.jpg"
+              alt="Макет A Plus"
+            />
+            <div class="product-item-hover">
+              <h4>A PLUS</h4>
+              <p>Лендинг курсов повышения квалификации</p>
+              <div>
+                <a href="#" aria-label="Цена" class="button">9900 РУБ.</a>
+              </div>
+            </div>
+          </article>
 
-        <article class="product-item">
-          <h3 class="visually-hidden">Макет кваст</h3>
-          <img
-            src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
-            alt="Хэдер браузера"
-          />
-          <img
-            src="../Nerds.web/img/catalog_img/img-kvast.jpg"
-            alt="Макет кваст"
-          />
-          <!-- <div class="product-item-hover">
-					<h4>КВАСТ</h4>
-					<p>Промо-сайт производителя крафтового кваса</p>
-					<div><a href="#" aria-label="Цена">9900 РУБ.</a></div>
-				</div> -->
-        </article>
-      </section>
+          <article class="product-item">
+            <h3 class="visually-hidden">Макет кваст</h3>
+            <img
+              class="browser-img"
+              src="../Nerds.web/img/catalog_img/catalog__img_svg/browser.svg"
+              alt="Хэдер браузера"
+            />
+            <img
+              src="../Nerds.web/img/catalog_img/img-kvast.jpg"
+              alt="Макет кваст"
+            />
+            <div class="product-item-hover">
+              <h4>КВАСТ</h4>
+              <p>Промо-сайт производителя крафтового кваса</p>
+              <div>

```

Comment:
for what did you use `<div>`?

Reply:
make a display: block for link instead of using div if it's possible

Reply:
idk )